host/default.yaml

```diff
@@ -384,6 +384,10 @@ spec:
         collectorName: "ps-high-load"
         command: "sh"
         args: ["-c", "ps -eo s,user,cmd | grep ^[RD] | sort | uniq -c | sort -nbr | head -20"]
+    - run:
+        collectorName: "ps-detect-antivirus-and-security-tools"
+        command: "sh"
+        args: [-c, "ps -ef | grep -E 'clamav|sophos|esets_daemon|fsav|symantec|mfend|ds_agent|kav|bdagent|s1agent|falcon|illumio' | grep -v grep"]
```

Comment:
I'd have `ps -ef | grep -v grep | ...` to ensure you exclude grep process from the output

Reply:
@banjoh I have it at the end of the command there to exclude it at the end. any reason it needs to be moved up? Not opposed to it. Just curious if there's something i'm missing